lib/TraackrApi/Influencers.php

```diff
@@ -81,6 +116,40 @@ public static function lookupTwitter($username, $type = 'USERNAME')
         return $inf->get(TraackrApi::$apiBaseUrl . 'influencers/lookup/twitter/' . $username, $parameters);
     }
 
+    /**
+     * Add social account
+     *
+     * @param array $p
+     * @return bool|mixed
+     * @throws MissingParameterException
+     * @throws \UnexpectedValueException
+     */
+    public static function addSocial($p = array())
+    {
+        $inf = new Influencers();
+
+        $p = $inf->addCustomerKey($p);
+        $inf->checkRequiredParams($p, array('platform', 'customer_key'));
+
+        // Validate business requirements
+        $platform = $p['platform'];
+        if ('TWITTER' !== $platform && 'INSTAGRAM' !== $platform) {
```

Comment:
Could move the checks up to the front of the method. No need to do any other work above it if they do not pass.

Reply:
i'd rather have them come in right after the checkRequiredParams() call. seems to be the norm we've settled on.